Tests/Controllers/PeopleController.cs

```diff
@@ -50,6 +51,13 @@ public Person PostPerson(string id, Person person)
         {
             person.Identifier = id;
             return person;
+        }
+
+        [HttpPost]
+        [Route("relationships/people")]
+        public JObject PostParams([FromBody] JObject bodyParams)
```

Comment:
Are you using `JObject` to mimic the dynamic nature of Ruby? If so, you might be better of just committing to type-safety in C#. Most (99%) of the time, you should be able to create a model class with the properties you want. Saule will already ignore properties that are not on the model.